docs/reference-implementation/quality-assurance.md

```diff
@@ -0,0 +1,99 @@
+- **Tools:** [Unit Testing Tools](#test-tools).
+
+---
+
+### Functional Testing
+
+
+- **Objectives:** Functional and end-to-end (E2E) testing ensures that each release behaves as expected according to defined epics and user stories.
+- **Requirements:** The Functional Requirements can be found [here](https://github.com/eu-digital-identity-wallet/eudi-wallet-product-roadmap/blob/main/EUDI%20Wallet%20Reference%20Implementation%20-%20Feature%20Map.md){:target="_blank"}.
+- **Test Specifications:** The Functional Testing Specifications can be found here:
+ - [Manual Testing](https://github.com/eu-digital-identity-wallet/eudi-doc-testing-application-internal/tree/main/src/test/resources/features){:target="_blank"}.
+ - [Automated Testing](https://github.com/eu-digital-identity-wallet/eudi-doc-testing-application-internal/tree/main){:target="_blank"}.
+- **Tools:** [Functional Testing Tools](#test-tools).
+
+---
+
+### Security Testing
+
+- **Objectives:** Security testing ensures the continuous security, integrity, and resilience of the EUDI Wallet applications throughout their lifecycle. By integrating automated and manual security testing within a Secure Software Development Life Cycle (SDLC) and aligning with OWASP MASVS and MASTG standards, the framework enables early detection and remediation of vulnerabilities. It combines code analysis, penetration testing, and vulnerability management using tools such as SonarQube, OWASP Dependency-Check, and DefectDojo to maintain compliance with recognized security standards and safeguard user trust in the EUDI Wallet ecosystem.
+- **Requirements and Test Specifications:** The Security Requirements and Security Test Specifications can be found [here](https://github.com/eu-digital-identity-wallet/eudi-doc-testing-application/blob/milestone/2025.Q4/doc/security/EUDI-Wallet-NiScy_Security_Requirements_and_Controls_Documentation.md){:target="_blank"}.
+- **Tools:** [Security Testing Tools](#test-tools).
+
+---
+
+### Performance Testing
+
+- **Objectives:** Performance tests to evaluate the performance of the EUDI Wallet application under average user load across various mobile devices. This testing focuses on CPU and memory usage, as well as network activity during common scenarios.
+- **Requirements:** The Performance Requirements can be found [here](https://github.com/eu-digital-identity-wallet/eudi-doc-testing-application/blob/milestone/2025.Q4/doc/performance/performance_requirements.md){:target="_blank"}.
+- **Test Specifications**: The Performance Test Specifications can be found [here](https://github.com/eu-digital-identity-wallet/eudi-doc-testing-application/blob/milestone/2025.Q4/doc/performance/performance_specifications.md){:target="_blank"}.
+- **Tools:** [Performance Testing Tools](#test-tools).
+
+## Test Tools
```

Comment:
Italics are not the best choice here

Why italics are not ideal
- Italics convey emphasis, not semantics. The tools listed (e.g., Manual, Serenity BDD, SQLCipher) are proper names, not text that needs visual emphasis.
- In technical documentation, italics can look subjective, as if certain tools are being highlighted without a clear reason.
- Italics do not support scannability. Readers skim for tool names, and italics add visual noise rather than structure.

Key principle
- Use formatting to signal meaning, not decoration. If a heading or term is important, structure should communicate that, not styling.